### ISSUE TITLE
Sending mail via a non-TLS relay

### ISSUE THREAD

**Issue by 7heo** [ccnmtl/hound#4]
I tried to use a non-TLS relay to send the mail alerts, and it failed every time. I then tried with postfix's "anonymous cipher only" feature, and it failed too.

The only way I made it work was by setting up a key and  cert in my postfix configuration. The source says it's supposed to work with both TLS and non-TLS connections, so I thought a bug report was in order.

**Comment by thraxil** [comment]
My usual setup for hound is running in a docker container that is linked to this one: https://github.com/ccnmtl/postfix-docker

That container basically just has a default ubuntu postfix install. I've never had to do anything more complicated to get that to work.

**Comment by 7heo** [comment]
Yes, I understand that it works on your machine. It does not on mine. Hence this ticket.

**Comment by 7heo** [comment]
Just to be clear: I created that issue because I spent quite some time trying to figure what would make the code work without TLS, and I failed. I'm not a go guru by any mean, so I figured that maybe you'd know.
